File: CommentaireVoyageur/src/modele/Commentaire.java
package modele;

import java.sql.Timestamp;

public class Commentaire
{
	protected int id;
	protected String titre;
	protected String auteur;
	protected String contenu;
	protected Timestamp date;
	protected int note;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getTitre() {
		return titre;
	}
	public void setTitre(String titre) {
		this.titre = titre;
	}
	public String getAuteur() {
		return auteur;
	}
	public void setAuteur(String auteur) {
		this.auteur = auteur;
	}
	public String getContenu() {
		return contenu;
	}
	public void setContenu(String contenu) {
		this.contenu = contenu;
	}
	public Timestamp getDate() {
		return date;
	}
	public void setDate(Timestamp date) {
		this.date = date;
	}
	public void setNote(int note) {
		this.note = note;
	}
	public int getNote() {
		return this.note;
	}
}

File: tikenix.me/model/Commentaire.php
<?php

class Commentaire{
	
	public $titre;
	public $auteur;
	public $contenu;
	public $date;
	public $note;
	
	/*function __construct($id, $titre, $auteur, $contenu, $date){
		$this->id = $id;
		$this->titre = $titre;
		$this->auteur = $auteur;
		$this->contenu = $contenu;
		$this->date = $date;
	}*/
	
	public function getTitre(){
		return $this->titre;
	}
	
	public function setTitre($titre){
		$this->titre = $titre;
	}
	
	public function getAuteur(){
		return $this->auteur;
	}
	
	public function setAuteur($auteur){
		$this->auteur = $auteur;
	}
	
	public function getContenu(){
		return $this->contenu;
	}
	
	public function setContenu($contenu){
		$this->contenu = $contenu;
	}
	
	public function getDate(){
		return $this->date;
	}
	
	public function setDate($date){
		$this->date = $date;
	}

	public function getNote(){
		return $this->note;
	}
	
	public function setNote($note){
		$this->note = $note;
	}
}
?>